.pylintrc

```diff
@@ -1,2 +1,5 @@
+[MASTER]
+load-plugins=pylint_protobuf
```

Comment:
@edgarriba this still doesn't work for me.  https://github.com/nelfin/pylint-protobuf/issues/19

Reply:
Looks like it may be caused by a recent version of protobuf: https://github.com/protocolbuffers/protobuf/issues/10372